instrumentation/lettuce/lettuce-5.0/src/test/groovy/LettuceReactiveClientTest.groovy

```diff
@@ -33,394 +34,394 @@ import static io.opentelemetry.auto.test.utils.TraceUtils.runUnderTrace
 import static io.opentelemetry.trace.Span.Kind.CLIENT
 
 class LettuceReactiveClientTest extends AgentTestRunner {
-  public static final String PEER_HOST = "localhost"
-  public static final String PEER_IP = "127.0.0.1"
-  public static final int DB_INDEX = 0
-  // Disable autoreconnect so we do not get stray traces popping up on server shutdown
-  public static final ClientOptions CLIENT_OPTIONS = ClientOptions.builder().autoReconnect(false).build()
-
-  @Shared
-  String embeddedDbUri
-
-  @Shared
-  RedisServer redisServer
-
-  RedisClient redisClient
-  StatefulConnection connection
-  RedisReactiveCommands<String, ?> reactiveCommands
-  RedisCommands<String, ?> syncCommands
-
-  def setupSpec() {
-    int port = PortUtils.randomOpenPort()
-    String dbAddr = PEER_HOST + ":" + port + "/" + DB_INDEX
-    embeddedDbUri = "redis://" + dbAddr
-
-    redisServer = RedisServer.builder()
-    // bind to localhost to avoid firewall popup
-      .setting("bind " + PEER_HOST)
-    // set max memory to avoid problems in CI
-      .setting("maxmemory 128M")
-      .port(port).build()
-  }
-
-  def setup() {
-    redisClient = RedisClient.create(embeddedDbUri)
-
-    println "Using redis: $redisServer.args"
-    redisServer.start()
-    redisClient.setOptions(CLIENT_OPTIONS)
-
-    connection = redisClient.connect()
-    reactiveCommands = connection.reactive()
-    syncCommands = connection.sync()
-
-    syncCommands.set("TESTKEY", "TESTVAL")
-
-    // 1 set + 1 connect trace
-    TEST_WRITER.waitForTraces(2)
-    TEST_WRITER.clear()
-  }
-
-  def cleanup() {
-    connection.close()
-    redisClient.shutdown()
-    redisServer.stop()
-  }
-
-  def "set command with subscribe on a defined consumer"() {
-    setup:
-    def conds = new AsyncConditions()
-    Consumer<String> consumer = new Consumer<String>() {
-      @Override
-      void accept(String res) {
-        conds.evaluate {
-          assert res == "OK"
-        }
-      }
+    public static final String PEER_HOST = "localhost"
```

Comment:
can you try reformatting this file in Intellij? i'm not sure why it's not using our `.editorconfig`

Reply:
reformat doesn't seem to change anything.